materials/08_statistical_analysis_all_stages.Rmd

```diff
@@ -0,0 +1,417 @@
+---
+title: Statistical analysis of all 3 cell cycle stages
+bibliography: course_refs.bib
+editor_options: 
+  chunk_output_type: console
+editor: 
+  markdown: 
+    wrap: 72
+---
+
+::: callout-tip
+#### Learning Objectives
+
+-   Using the `limma` package (@Smyth2004), design a statistical model to test for
+    differentially abundant proteins between more than two conditions
+:::
+
+```{r, echo = FALSE, fig.align = "center", out.width = "90%"}
+knitr::include_graphics("figs/flow_chart/flow_chart.010.png", error = FALSE)
+```
+
+
+::: callout-note
+This workflow is an adjunct to the [Statistical analysis](./06_statistical_analysis.html) section which demonstrates how to perform single comparison statistical with `limma` (@Smyth2004). Please first read through that material, since it includes background explanations and clarifications which are not repeated here.
+:::
+
+```{r, eval=TRUE, include=FALSE}
+library("QFeatures")
+library("NormalyzerDE")
+library("limma")
+library("factoextra")
+library("org.Hs.eg.db")
+library("clusterProfiler")
+library("enrichplot")
+library("patchwork")
+library("tidyverse")
+library("pheatmap")
+library("here")
+
+load(here("materials/output/lesson04.rda"), verbose = TRUE)
+```
+
+## Differential expression analysis
+
+Having cleaned our data, aggregated from PSM to protein level, completed
+a log2 transformation and normalised the data, we are now ready to carry
+out statistical analysis. 
+
+The aim of this section is to answer the
+question: *"Which proteins show a significant change in abundance
+between the multiple cell cycle stages?"*.
+
+Our null hypothesis is: **(H0)** The change in abundance for a protein
+between cell cycle stages is zero.
+
+Our alternative hypothesis is: **(H1)** The change in abundance for a
+protein between cell cycle stages is not zero.
+
+
+We will use `limma` to perform the statistical tests using an **empirical Bayes-moderated linear model**. Please see the [Statistical analysis](./06_statistical_analysis.html) section for more details about `limma` and its application in quantitative proteomics. 
+
+Depending on whether the linear model is used to perform single comparisons or multifactorial comparisons, the test statistic will either be a
+t-value or an F-value, respectively. Here, we are performing a multifactorial comparison, so we will obtain an F-value. 
```

Comment:
Well, I guess you will do both. Because when you extract the pairwise comparisons you get a t-value.
Maybe we could say:

```suggestion
t-value or an F-value, respectively. Here, we will perform both, starting with assessing the overall effect of cell cycle.
```

Reply:
Good point indeed!